sqlitecluster/SQLite.h

```diff
@@ -192,6 +197,8 @@ class SQLite {
     // Enable/disable SQL statement tracing.
     static atomic<bool> enableTrace;
 
+    string _lastQuery;
```

Comment:
does the underscore indicate this should be private? in any case, we prefer not prefixing with underscore for privates (that's an old thing -- @tylerkaraszewski correct me if I'm wrong)

Reply:
We don't use underscores for private variables in auth but we still do in bedrock.

Reply:
is it not? my bad... gonna fix it